### DISCUSSION TITLE
Questions regarding dBR/dq^2 distribution for B+ ->μμΚ+ decay

### DISCUSSION THREAD

**Discussion by gmelachr** [flav-io/flavio#284] [Q&A]
Dear Mr. Straub, all, 

my name is Georgios Melachroinos. I am a PhD Student at the university of Athens, Greece, and I am a member of team on the CMS experiment working on rare B decays. For the muon channel, we have adequate statistics for a differential measurement vs q^2: BR(B->μμΚ; q2 bin_i)/ BR(B->J/ψΚ->μμΚ), and we are thus looking for a differential theoretical prediction for comparison with data. In the most recent publication by LHCb ([https://arxiv.org/pdf/2212.09153.pdf](https://mmm.cern.ch/owa/redir.aspx?C=eUmmNY-GNofihW28ncZXEpZDbPYLLJUonKzPdAKhJAIy28TUXfLaCA..&URL=https%3a%2f%2farxiv.org%2fpdf%2f2212.09153.pdf)) underneath figure 3, they mention that they are using the flavio package [https://arxiv.org/pdf/1810.08132.pdf](https://mmm.cern.ch/owa/redir.aspx?C=80nzFT2MvJhg_b1HlKmg1cBI7FEI1eB61yYH2hdVLsFCAsXUXfLaCA..&URL=https%3a%2f%2farxiv.org%2fpdf%2f1810.08132.pdf). 

As a first attempt, we have plotted dBR(B->μμΚ)/dq2 — as shown in the png file. The low q2 (in LHCb most recent paper it is mentioned as central q2 bin) and the high q2 bins (1.1-6.0 GeV^2 and 16.0-23 GeV^2 respectively), are both marked in green. The J/ψ and ψ(2S) bins are marked in red [the J/ψ bin covers 2.9-3.2 GeV (q^2: 8.41-10.24) and the ψ(2S) bin covers 3.55-3.8 GeV (q^2: 12.6025 - 14.44 GeV^2)]. There are two small “triangular" excesses, the first in the J/ψ bin and the second above the J/ψ but below the ψ(2S) resonance. We suspect that the first blip is due to interference with the Β+->J/ψΚ+ mode. We would then expect that the second triangular excess would be from interference with the Β+->ψ(2S)K+ mode, however it is shifted to lower q^2 values. We presume that you have already observed this type of small “blip” on these plots? And if yes, do you have an explanation for them? If not, could you point us to whoever is responsible for the code generating the B—>μμΚ Branching fraction? 

Many thanks in advance, 
Georgios Melachroinos

![theoretical_uncertainties](https://user-images.githubusercontent.com/58115516/211358098-f4875d99-4fff-4e3a-9b14-1b34134d0c37.png)

**Comment by peterstangl** [comment]
Hi Georgios,

it's great to hear that CMS is working on this analysis! 

You should actually have received the following warning when you did the above plot:
```
UserWarning: The predictions in the region of narrow charmonium resonances are not meaningful
```
The prediction does *not* include resonance contributions from the $J/\Psi$ or the $\Psi(2S)$. I think the "excesses" that you see in this region are actually just artifacts of the fact that virtual corrections to the matrix elements are implemented for the low q2 and the high q2 regions in terms of separate functions. The "excesses" then corresponds to the transition from the functions valid at low q2 to the functions valid at high q2 (which is not continuous, resulting in this kind of kink). This region of narrow charmonium resonances should be excluded since the theory prediction there is really not meaningful.

**Comment by gmelachr** [comment]
Dear Mr. Stangl,

Thank you very much for your answer, which is really helpful. 

Indeed, we do receive this warning during the plotting.  We did not know that these kinks are due to different functions that are used in the low and in the high q2 bins. Is there a reference for this? Any pointer would be much appreciated.

Taking advantage of this communication we would like to ask you a couple more questions?
1. While we understand that "This region of narrow charmonium resonances should be excluded since the theory prediction there is really not meaningful.”:, the question is how do we include the interference effects [see for example this paper from LHCb (https://arxiv.org/pdf/1612.06764.pdf) .  A related question is what to do with the resonances beyond the J/ψ and ψ(2S), e.g. the  ψ(4160).  Is the prediction above the ψ(2S) reliable?

2. Integrating the plot from Flavio to obtain the total Branching fraction for μμΚ we obtain (5.98±0.89)x10^-7 while the PDG value is (4.37±0.27)×10^-7. [[https://pdg.lbl.gov/2022/listings/contents_listings.html](https://mmm.cern.ch/owa/redir.aspx?C=EwT1jEhHgAf3-W_caa6d6zQDMCc6pGhK8htb0YaWi_hy306awPPaCA..&URL=https%3a%2f%2fpdg.lbl.gov%2f2022%2flistings%2fcontents_listings.html)]. Do you happen to know if this disagreement is due to the inclusion of the LHCb measurement in the PDG value?  Or is it a general disagreement between theory and experiment [e.g. the LHCb measurements have been lower than theory].

Thank you very much in advance,
On behalf of the CMS RK Analysis WG,
Giorgos

**Comment by peterstangl** [comment]
> Indeed, we do receive this warning during the plotting. We did not know that these kinks are due to different functions that are used in the low and in the high q2 bins. Is there a reference for this? Any pointer would be much appreciated.

The two kinks arise as follows:
- The kink at $q^2=4 m_c^2$ is due to the one-loop contributions of four-quark operators $O_1,...,O_6$ to the $\langle s\ell^+\ell^-|O_i|b\rangle$ matrix elements. These contributions are taken into account in an effective $q^2$-dependent Wilson coefficient $C_9^{\rm eff}(q^2)$ in terms of a function $Y(q^2)$, which is e.g. given in Eq. (2.10)  of [arXiv:0811.1214](https://arxiv.org/abs/0811.1214). $Y(q^2)$ depends on another function $h(q^2, m)$ (given in Eq. (2.11) of [arXiv:0811.1214](https://arxiv.org/abs/0811.1214)) that has a discontinuity at $q^2=4 m^2$.
- The second kink is due to $\alpha_s$-corrections to the matrix elements of four-quark operators. They have been calculated analytically for the low $q^2$ region in [arXiv:hep-ph/0109140](https://arxiv.org/abs/hep-ph/0109140) and for the high $q^2$ region in [arXiv:0810.4077](https://arxiv.org/abs/0810.4077). The latter reference provides both results in terms of a C++ code that is valid in both regions. The transition between the results valid at low and high $q^2$ leads to a discontinuity at $q^2=0.4\ m_b^2$.
 
> While we understand that "This region of narrow charmonium resonances should be excluded since the theory prediction there is really not meaningful.”:, the question is how do we include the interference effects [see for example this paper from LHCb (https://arxiv.org/pdf/1612.06764.pdf) .

As far as I know there is no straight forward way to include the interference effects. The LHCb paper basically uses an ad-hoc model for the resonances that might work to some extend but it is difficult to judge its general validity. In flavio, the unknown "non-factorizable" effects that cannot be described by local matrix elements are parameterized outside of the region of narrow charmonium resonances as described in appendix B.1 of [arXiv:2103.13370](https://arxiv.org/abs/2103.13370), i.e. these effects are taken into account as uncertainties of the theory predictions. See also the related discussion in section 3.1 of [arXiv:1503.05534](https://arxiv.org/abs/1503.05534). An alternative could be the approach of [arXiv:2206.03797](https://arxiv.org/abs/2206.03797).

> A related question is what to do with the resonances beyond the J/ψ and ψ(2S), e.g. the  ψ(4160).  Is the prediction above the ψ(2S) reliable?

Above the narrow charmonium resonances, the theory predictions are only valid for sufficiently global, i.e. $q^2$ integrated, observables, see [arXiv:1101.5118](https://arxiv.org/abs/1101.5118). In practice this means that comparisons between theory and experiments in the high $q^2$ region should use the widest possible bins. This also means that a differential prediction that can be compared to a differential measurement is not really available in this region.

> Integrating the plot from Flavio to obtain the total Branching fraction for μμΚ we obtain (5.98±0.89)x10^-7 while the PDG value is (4.37±0.27)×10^-7. [[https://pdg.lbl.gov/2022/listings/contents_listings.html](https://mmm.cern.ch/owa/redir.aspx?C=EwT1jEhHgAf3-W_caa6d6zQDMCc6pGhK8htb0YaWi_hy306awPPaCA..&URL=https%3a%2f%2fpdg.lbl.gov%2f2022%2flistings%2fcontents_listings.html)]. Do you happen to know if this disagreement is due to the inclusion of the LHCb measurement in the PDG value?  Or is it a general disagreement between theory and experiment [e.g. the LHCb measurements have been lower than theory].

The PDG value that you are quoting is just the result of the LHCb analysis you mentioned above, where an ad-hoc model for the resonances has been used in order to obtain a "non-resonant" total branching fraction. Simply integrating the flavio prediction over the full $q^2$ range is not the same as what they do and one cannot expect to get the same result. In particular, the integration over the narrow charmonium region might contain some artifacts of functions that are actually not meaningful there (as e.g. the those functions that are responsible for the two kinks you observed).
However, you might know that also the binned $B\to K\mu\mu$ branching ratios measured by LHCb ([arXiv:1403.8044](https://arxiv.org/abs/1403.8044)) show deviations from Standard Model predictions. So even disregarding the issues in the region of narrow charmonium resonance there are disagreements between LHCb measurements and theory predictions.

**Comment by gmelachr** [comment]
Dear Mr. Stangl, 

we would like to thank you very much for your answers. Your comments were very helpful and we appreciate your willingness to explain everything. Thank you very much!  

on behalf of the CMS RK Analysis WG, 
Giorgos Melachroinos
